src/lib/holocene/table/paginated-table/paginated.svelte

```diff
@@ -26,10 +26,10 @@
   export let nextPageButtonLabel: string;
   export let previousPageButtonLabel: string;
   export let maxHeight = '';
+  export let pageSizeOptions: string[] = options;
```

Comment:
How do you feel about updating the options here to have a 50 page size default? It makes things load pretty noticeably faster in my testing